Sources/Web3Auth/Provider.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-public enum Web3AuthProvider: String, Codable {
+public enum Web3AuthProvider: String, Encodable {
```

Comment:
please just use codable.. it includes encodable and decodable.!
We need it like that for flutter

Reply:
Done

Reply:
done

Reply:
Make sure you have inits for this.

Reply:
don't need to provide init for enums